src/components/cart/proceed-payment/index.tsx

```diff
@@ -0,0 +1,242 @@
+import styled from "styled-components";
+import { colors, sizes, fontFamilies, letterSpacing } from "../../../variables";
+import Searchpath from "../../common/Searchpath";
+import data from "../../common/constants.json";
+import Label from "../../common/label";
+import { useForm, Controller } from "react-hook-form";
+import Unselected from "../../../assets/common/Radio button/radio_button_unselected.png";
+import InputField from "../../common/textbox";
+
+const ProceedPayment = () => {
+  const { register, handleSubmit, control, reset } = useForm();
+  const [cartItems, setCartItems] = useState(data.cartContents.myCart);
+  const [radioButtons, setRadioButtons] = useState({
+    userCode: false,
+    bestOffers: true,
+  });
+  const handleClick = (e: React.MouseEvent<HTMLElement>) => {
+    setCartItems([]);
+  };
+  const handleRadioClick = (e: React.MouseEvent<HTMLInputElement>) => {
+    if (e.currentTarget.id === "userCode" && radioButtons.userCode === true) {
+      setRadioButtons({ userCode: false, bestOffers: false });
+    } else if (e.currentTarget.id === "userCode") {
+      setRadioButtons({ userCode: true, bestOffers: false });
+    } else if (
+      e.currentTarget.id === "bestOffer" &&
+      radioButtons.bestOffers === true
+    ) {
+      setRadioButtons({ userCode: false, bestOffers: false });
+    } else if (e.currentTarget.id === "bestOffer") {
+      setRadioButtons({ userCode: false, bestOffers: true });
+    } else {
+      return;
+    }
+  };
+  return (
+    <PageSection>
+      <Searchpath path={data.searchPath.menu1} className="grey-dark" />
+      <ContentContainer>
+        <ProgressSection></ProgressSection>
+        <CartSection>
+          <StepsSection>
+            <StepTitle content={data.cartTexts.title} />
+            <CookingInstructionTitle>
+              {data.cartData.cookingInstruction}
+            </CookingInstructionTitle>
+            <InputField
+              name={data.cartData.textboxLabel}
+              isPassword={false}
+              style={{
+                width: "263px",
+                margin: "auto",
+                color: `${colors.grey_4a4a4a}`,
+              }}
+            />
+          </StepsSection>
+          <PaymentContainer>
+            <RadioTitle>{data.offersRadio.title}</RadioTitle>
+            <Wrapper>
+              <RadioInput
+                type="radio"
+                id="userCode"
+                name="referal"
+                value="userCode"
+                onClick={handleRadioClick}
+                checked={radioButtons.userCode}
+              />
+              <OptionName htmlFor="userCode">
+                {data.offersRadio.options[0]}
+              </OptionName>
+            </Wrapper>
+            {radioButtons.userCode === true ? (
+              <UserCodeSection>
+                <InputField
+                  name="Add your promo code"
+                  isPassword={false}
+                  style={{ width: "238px", height: "33px" }}
```

Comment:
Remove style props and use style component here

Reply:
Styling method made same as label and used styled component.